docs/client/initializing.md

```diff
@@ -0,0 +1,81 @@
+# <img align="center" src="../images/logo.png">  Initializing Your Java Client
+
+The first step to using your generated client in code is to import and initialize your client. Our SDKs are modelled such
+that the client is the main point of access to the generated code.
+
+## Importing Your Client
+
+You include the package from the namespace specified when generating (under flag `--namespace`). For the sake of this example,
+let's say the namespace is `com.azure.pets`.
+
+Putting this together, we import our client like so:
+
+```java
+package com.azure.pets;
+```
+
+## Initializing and Authenticating Your Client
+
+Your client's name is detailed in the swagger, (TODO link to swagger docs), and let's say
+ours is called `PetsClient`. We use builders to build our client, with each builder being the service client's name + suffix `Builder()`.
+The builder takes in any number of parameters, and ultimately calls `buildClient()`, which ultimately what returns the built client.
+
+```java
+package com.azure.pets;
+
+public static void main(String args[])
+{
+    PetsClient client = new PetsClientBuilder().buildClient();
+}
+```
+
+You can also install your client with a credential, using the flag `--credential-types`. If you generate with `--credential-types=TokenCredential`, your
+client will take in an [Azure Active Directory (AAD) token credential][aad_authentication]. We always recommend
+using a [credential type][identity_credentials] obtained from the [`com.azure.identity`][azure_identity_library] package for AAD authentication. For this example,
+we use the most common [`DefaultAzureCredential`][default_azure_credential].
+
+As an installation note, the [`com.azure.identity`][azure_identity_library] is not listed as a dependency in the `pom` we generate
+(see `--regenerate-pom` in our [flag index][flag_index] for more information), so you would need to install this library separately.
```

Comment:
@jianghaolu What does `regenerate-pom` intend to do? Does it decide whether pom.xml is generated or not? If yes, I could use this in fluent to skip pom in Lite.

I think currently data-plane does not generate pom.xml.

Reply:
For now we've only used this flag for fluent lite and it overwrites the current pom if there's any.